### ISSUE TITLE
Specific ObjectId in "Add-AzureADGroupMember"

### ISSUE THREAD

**Issue by tdannecy** [itmanagerguy/autopilot-intune#1]
Could you provide a bit more information on which object you're referencing here?

Is this a typo, or is it referencing a built-in AAD group/object?

https://github.com/itmanagerguy/autopilot-intune/blob/f05def3cc4f2691f455aae042e9088da68b3c678/AutopilotImport.ps1#L164

**Comment by itmanagerguy** [comment]
You'll see in the above lines it's referencing a group we had that we'd add
people to when we Autopiloted their machines. That section of 5 or so lines
could be removed if not needed for your specific needs.

This script is a bit outdated now, and I need to update it. Will be doing
so soon.

**Comment by tdannecy** [comment]
Right. Instead of using the typed out ObjectId and RefObjectId here on line 164, you should call `userobjectid` and `groupobjectid` that you defined on lines 161 and 162.

**Comment by itmanagerguy** [comment]
Right. A bit of a miss on my end, as I had modified this for our internal
use, and tried to cleanse it for GitHub usage and must have missed that
piece.

Thanks for the catch
